modules/images/image-loading-optimization/storage/lock.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Metrics storage lock.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gets the TTL (in seconds) for the page metric storage lock.
+ *
+ * @since n.e.x.t
+ * @access private
+ *
+ * @return int TTL in seconds, greater than or equal to zero.
```

Comment:
Based on https://github.com/WordPress/performance/pull/878#discussion_r1393429581:
```suggestion
 * @return int TTL in seconds, greater than or equal to zero. A value of zero means that the storage lock should be disabled and thus that transients must not be used.
```